docs/product-editor-development/product-editor-extensibility-guidelines.md

```diff
@@ -0,0 +1,97 @@
+
+* **General**: Essential product information, including the name, image, and description. This tab is also where key features live for non-standard product types: downloads, groups, links, etc.
+
+* **Organization**: This tab contains all the data used to organize and categorize product information: from categories to attributes. Best for extensions that provide new ways to describe products, e.g., product identifiers, statuses, special tags, etc.
+
+* **Pricing**: List price, sale price, and tax options. Best for extensions that allow merchants to set up additional payment methods (e.g., Subscriptions) or add advanced pricing schemes, like wholesale.
+
+* **Inventory**: Basic inventory settings and stock management options. Here merchants come to update the quantity at hand or mark the product as out of stock. Best for extensions that enable conditional inventory management, dropshipping options, or various restrictions.
+
+* **Shipping**: All the information merchants need to enter to present customers with accurate shipping rates at checkout. Best for physical product details that may impact shipping (e.g. capacity or volume), additional shipping carrier settings, or custom shipping options.
+
+* **Variations**: Contains product options and variations. It is the only tab that doesn’t support extensibility in its current form (you can add your extension to the variation details page as well as provide custom update actions triggered from the table view or the bulk editor).
+
+## Where should your extension go?
+
+Depending on the type of your extension (and your use case), you can pick an interface location that best suits how users will interact with it.
+
+To choose, put yourself in the merchant’s shoes: where would you go to find this feature? What is it related to? Adding your extension to a group of similar features will help merchants navigate the form and make it easier for them to find your extension.
+
+### Field 
+
+Field extensions are the simplest type of extension. They can add new, replace, and expand existing form fields in any section, hide fields, and change form defaults.
+
+#### ✅ What they are for
+
+* Single-field, supplementary features
+* Showing or hiding form elements depending on specific conditions
+
+#### ❌ What they aren’t for
+
+* Multi-field or multi-step forms
+* Complex tables
+
+#### Use cases
+
+* Adding extra product details, f.e. volume under Shipping
+* Entering custom data, f.e. color or date and time
+* Selecting from a third-party system, f.e. warranty type
+
+### Group 
+
+Groups add extra fields to existing form sections. They are small forms with a low to medium level of complexity. This interface location works best for extensions that add extra features that build off an existing Woo functionality.
```

Comment:
These are simply nested Sections in the API. We should not refer to them as Groups, since those are the top-level tabs.